hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/replicas/ReplicasVerify.java

```diff
@@ -207,15 +271,95 @@ void processKey(OzoneClient ozoneClient, String volumeName, String bucketName, S
     }
 
     keyNode.put("pass", keyPass);
-    if (!keyPass) {
+    if (keyPass) {
+      keysPassed.incrementAndGet();
+    } else {
+      keysFailed.incrementAndGet();
       allKeysPassed.set(false);
+      
+      for (String failedType : failedVerificationTypes) {
+        AtomicInteger counter = failuresByType.get(failedType);
+        if (counter != null) {
+          counter.incrementAndGet();
+        } else {
+          failuresByType.computeIfAbsent(failedType, k -> new AtomicInteger(0)).incrementAndGet();
+        }
+      }
     }
 
     if (!keyPass || allResults) {
       keysArray.add(keyNode);
     }
   }
 
+  /**
+   * Adds ShutdownHook to print summary statistics.
+   */
+  private void addShutdownHook() {
+    ShutdownHookManager.get().addShutdownHook(() -> {
+      if (endTime == 0) {
+        endTime = System.nanoTime();
+      }
+      printSummary(System.out);
```

Comment:
We should print the summary to stderr so the json output can be redirected into tools like jq and does not need to be manually trimmed off. Even though it is not technically an "error", this pattern is used with standard commands like `time`, which prints its execution time summary to stderr to avoid interfering with the output of the command it was timing.
```suggestion
      printSummary(System.err);
```